src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,67 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { useState } from 'react';
+import { FilterStatus } from '../../types/FilterStatus';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface TodoFilterProps {
+  onFilterChange: (value: string) => void;
+  onSelectedChange: (value: any) => void;
```

Comment:
i think the type here would be some strange event, you could add it but sometimes it's not even worth it, nice